src/rpcblockchain.cpp

```diff
@@ -433,14 +433,18 @@ Object blockToJSON(const CBlock& block, const CBlockIndex* blockindex, bool fPri
     result.push_back(Pair("NeuralHash",bb.NeuralHash));
     if (bb.superblock.length() > 20)
     {
-        //result.push_back(Pair("SuperblockLength", RoundToString((double)bb.superblock.length(),0) ));
         //12-20-2015 Support for Binary Superblocks
         std::string superblock=UnpackBinarySuperblock(bb.superblock);
         std::string neural_hash = GetQuorumHash(superblock);
         result.push_back(Pair("SuperblockHash", neural_hash));
-        result.push_back(Pair("SuperblockLength", (double)bb.superblock.length()));
+        result.push_back(Pair("SuperblockLength", (long)superblock.length()));
```

Comment:
This changes the meaning of "SuperblockLength". Will this cause any compatibility issues?